testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -195,7 +196,7 @@ class MiniAppDisplayActivity : BaseActivity() {
                     miniAppId: String,
                     accessTokenScope: AccessTokenScope,
                     onSuccess: (tokenData: TokenData) -> Unit,
-                    onError: (message: String) -> Unit
```

Comment:
Then, we need to update the user-guide too:
<img width="467" alt="Screen Shot 2021-05-25 at 0 03 10" src="https://user-images.githubusercontent.com/66930373/119367190-a87cff80-bcec-11eb-8cc9-9bcf82018736.png">

Reply:
yes we need to update it @khairul-alam-licon

Reply:
I think you still need to update `USERGUIDE.md` @munir-rakuten :)